doc/guides/writing-tests.md

```diff
@@ -209,6 +209,35 @@ const assert = require('assert');
 const freelist = require('internal/freelist');
 ```
 
+### Assertions
+
+When writing an assertion for comparison, prefer the strict versions:
```

Comment:
You can simply drop `for comparison`.. "When asserting, prefer strict versions" - How does it sound?